### DISCUSSION TITLE
RS485 on different pi ?

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#713] [App and O/S Config Questions]
I've had njsPC on a pi in an equipment closet inside that house.  It shares a RS485 connection to Pentair equipment in the pool area.

Now I want to add a DIY chem controller.  So, a pi needs to be in the pool equipment area to control the dosing pumps.

Do I really need the RS485 connection to just run REM for the chem controller?  If so, do I need to physically wire the RS485 to the pi I'm going to use for chem controller, or can I forward data from the inside pi (that's connected to the RS485).

**Comment by davthomaspilot** [comment]
So, I think that was a stupid question.  Nothing in the REM documentation says you need an RS485.

I was thinking the REM was part of njsPC--not a standalone app.

I just installed it and can play with the web interface.

Looks fantastic.   The page to view and change RPI GPIO itself is awesome.  Can't wait until the hardware gets here!

**Reply by rstrouse** [reply]
Yes it was designed to be distributed.  You can run njsPC, dashPanel, and REM on separate devices if you like.
